app/Console/Commands/Wiki/OrgDigitalBibleLibraryCompare.php

```diff
@@ -62,7 +61,8 @@ private function organizationExists($dbl_org)
     {
         $translationMatchExists    = OrganizationTranslation::where('name', $dbl_org->full_name)->first();
         $relationshipAlreadyExists = OrganizationRelationship::where('organization_parent_id', $this->dbl_id)
-                                                             ->where('relationship_id', $dbl_org->id)->first();
+                                                            //  ->where('relationship_id', $dbl_org->id)->first();
```

Comment:
why is this where clause commented out? Why did this even come up as an issue? I think this file should probably be deleted at some point

Reply:
I think it should be removed because the relationship_id column doesn't exist in the current organization_relationships entity. If it is ok to remove it, I update the OrgDigitalBibleLibraryCompare.php file in the next PR:

- https://github.com/faithcomesbyhearing/dbp/pull/474